server/app/main.py

```diff
@@ -91,41 +91,57 @@ async def display_exec(request: Request, pipeline_name: str):
 @app.get("/display_lineage/{pipeline_name}", response_class=HTMLResponse)
 async def display_lineage(request: Request, pipeline_name: str):
     # checks if mlmd file exists on server
-    print(os.getcwd())
+    img_path=""
     if os.path.exists(server_store_path):
         query = cmfquery.CmfQuery(server_store_path)
         if (pipeline_name in query.get_pipeline_names()):
-            query_visualization(server_store_path, pipeline_name)
-            print(pipeline_name)
+            if os.path.exists("data/mlmd_for_img/mlmd"):
+                os.remove("data/mlmd_for_img/mlmd")
```

Comment:
This is highly suboptimal ..we are converting to json then to mlmd ..then doing the visualization. why are we not doing the visualization on the mlmd present. this will not scale